docs/rfc/rfc-6.adoc

```diff
@@ -0,0 +1,228 @@
+
+The responsibility of executing the second group of transactions should be
+delegated to willing maintainers. Given that the preparation of the input data
+in some cases is complicated and may require access to the Bitcoin chain, the
+maintainer functionality should be built-in into the client and activated in the
+configuration. Eventually, we may decide to use networks like Gelato but given
+that the ECSDSA wallet could be slashed on the redemption proof timeout, such
+functionality should also be implemented in the client so that the client can
+protect the stake if required. Last but not least, Gelato Polywrap resolver
+allowing assembling more complicated transaction inputs is not yet available and
+we do not want to block the release on this functionality. Maintainers will be
+reimbursed in ETH and may receive an additional reward for performing their
+duties.
+
+Transactions belonging to the third type can be executed by anyone and all
+off-chain clients will be responsible for submitting them. No ETH will be
+reimbursed and T coming from slashed stakes will be the only optional reward.
+No submission order will be maintained and all clients will try to report
+misbehavior at the same time. It is more important to keep the network healthy
+and free from misbehavior than to optimize costs.
+
+==== Group member incentives
+
+[[submit-dkg-result]]
+===== Submit + approve DKG result
+
+No submission order will be enforced on-chain and the transaction submitter will
+have ETH cost reimbursed. To disincentivize front-running, no additional reward
+will be released by this transaction. Off-chain clients should have a
+gentleman's agreement to follow the submission order based on
+`hash(new_group_pubkey) % group_size`.
+The submission order will be unenforceable but everyone will defaultly
+configured to follow the order anyway.
+
+The first member responsible for submitting the DKG result is a member with
+index `hash(new_group_pubkey) % group_size`. If that member did not submit the
+result,  `(hash(new_group_pubkey) % group_size) + 1` becomes responsible next.
+Group members who were inactive and skipped their responsibility might be
+reported by the wallet as ones who failed the heartbeat and marked as ineligible
+for rewards.
+
+For example, if `hash(new_group_pubkey) % group_size = 62`, `group_size = 64`,
+and member `9` submitted DKG result, members with indexes
+`{62, 63, 64, 1, 2, 3, 4, 5, 6, 7, 8}` might be marked as ineligible for
+rewards. Inactive group members are determined once the result is submitted and
+accepted by the chain.
+
+Given that the DKG result submission process consists of two transactions:
+submitting the result and approving the result after the challenge period ends,
+in order for the operator to be considered active, they must submit **both**
+transactions.
+
+===== Report heartbeat failed
+
+A wallet should periodically execute off-chain heartbeats and report inactive
```

Comment:
> ..should periodically execute off-chain heartbeats..

Do we know what is "periodically" or what action should trigger off-chain heartbeats? Can you please give an example?

Reply:
https://github.com/keep-network/tbtc-v2/blob/main/docs/rfc/rfc-1.adoc#heartbeat